### PR TITLE
Init clerk webhook

### PR DESCRIPTION
**Pull request by thibaultleouay**

### CODE REVIEW

File: apps/web/src/app/api/ping/route.ts
Comment (mxkaske) [inline apps/web/src/app/api/ping/route.ts:1]: Why not using the `/api/v0/ping` route?

File: apps/web/src/middleware.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { authMiddleware } from "@clerk/nextjs";
 
 export default authMiddleware({
-  publicRoutes: ["/", "/api/og"],
+  publicRoutes: ["/", "/api/og", "/api/webhook/ping", "/api/webhook/clerk"],
```
Comment (mxkaske) [inline apps/web/src/middleware.ts]: We will need to check for placeholders here. I’m doing the same currently

File: packages/api/src/router/clerk/type.ts
Comment (mxkaske) [inline packages/api/src/router/clerk/type.ts:1]: Nice!

**Review by mxkaske — COMMENTED** [review]
🚀

File: apps/web/src/middleware.ts (hunk 1 above)
Comment (thibaultleouay) [reply apps/web/src/middleware.ts]: I think we need to do with the matcher instead for the middle ware ?

**Pull request merged**